src/proposal/Proposal.sol

```diff
@@ -48,6 +48,10 @@ contract Proposal is IProposal, OwnableUpgradeable, ModuleManager {
         _;
     }
 
+    // TODO: We don't use this modifier or the manager() function
```

Comment:
Cant we remove owner already?
If not move this comment into #281 and delete it here

Reply:
I'd do that in a separate PR. removed the comment for now